src/components/number-of-films/number-of-films.js

```diff
@@ -0,0 +1,9 @@
+import './number_of_films.css'
+
+function NumberOfFilms () {
+    return (
+        <div className={'number_of_films'}><span className={'number'}>39</span> movies found</div>
```

Comment:
```suggestion
        <div className='number_of_films'><span className='number'>{moviesCount}</span> movies found</div>
```
Maybe it would be better to pass the number of movies like props?

Reply:
Done